build/azDevOps/azure/air-infrastructure-data.yml

```diff
@@ -106,154 +106,102 @@ stages:
                     TF_VAR_name_domain: $(domain)
                     TF_VAR_stage: $(Environment.ShortName)
 
-  # yamllint disable rule:comments-indentation
-  # - stage: Prod
-  #   dependsOn: Build
-  #   condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main'))
-  #   variables:
-  #     - group: amido-stacks-infra-credentials-prod
-  #     - group: stacks-credentials-prod-kv
-  #     - name: Environment.ShortName
-  #       value: prod
-  #     - name: dns_base_domain # Available in Terraform Output, but not yet enabled to export as pipeline vars
-  #       value: prod.amidostacks.com
-  #     - name: core_environment
-  #       value: prod
-  #   jobs:
-  #     - deployment: AppInfraProd
-  #       pool:
-  #         vmImage: $(pool_vm_image)
+ #   yamllint disable rule:comments-indentation
```

Comment:
I'm  not sure we need this if we move the comments below to above the lines?

Reply:
thanks for the help, all updated now.